issue.go

```diff
@@ -960,3 +969,65 @@ func (s *IssueService) Delete(issueID string) (*Response, error) {
 	resp, err := s.client.Do(req, nil)
 	return resp, err
 }
+
+// GetWatchers wil return all the users watching/observing the given issue
+func (s *IssueService) GetWatchers(issueID string) (*[]User, *Response, error) {
+	watchesApiEndPoint := fmt.Sprintf("rest/api/2/issue/%s/watchers", issueID)
+
+	req, err := s.client.NewRequest("GET", watchesApiEndPoint, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	watches := new(Watches)
+	resp, err := s.client.Do(req, watches)
+	if err != nil {
+		return nil, nil, NewJiraError(resp, err)
+	}
+
+	result := []User{}
+	user := new(User)
+	for _, watcher := range watches.Watchers {
+		user, resp, err = s.client.User.Get(watcher.Name)
+		if err != nil {
+			return nil, resp, NewJiraError(resp, err)
+		}
+		result = append(result, *user)
+	}
+
+	return &result, resp, nil
+}
+
+// SetWatcher adds watcher to the given issue
```

Comment:
Can you please add a link to the respective documentation here?
See the godoc of other API calls as example